frame/session/benchmarking/Cargo.toml

```diff
@@ -13,6 +13,7 @@ readme = "README.md"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
+scale-info = { version = "0.9.0", default-features = false, features = ["derive"] }
```

Comment:
Missing `scale-info/std` feature for this Cargo.toml?

Reply:
Ditto, removed dependency entirely